ros2_ws/config/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
I assume you didn't mean to add this file to this PR?

Reply:
Yes of course, thanks